app/controllers/api/v1/hosts_controller.rb

```diff
@@ -30,7 +30,8 @@ def show
         param :location_id, :number, :required => true, :desc => "required if locations are enabled" if SETTINGS[:locations_enabled]
         param :organization_id, :number, :required => true, :desc => "required if organizations are enabled" if SETTINGS[:organizations_enabled]
         param :environment_id, String, :desc => "required if host is managed and value is not inherited from host group"
-        param :ip, String, :desc => "not required if using a subnet with DHCP proxy"
+        param :ip, String, :desc => "IPv4 address, not required if using a subnet with DHCP proxy"
+        param :ip6, String, :desc => "IPv6 address"
```

Comment:
As of now there is (unfortunately) no DHCPv6 support anywhere in Foreman. Hosts with a DNS-enabled domain without IPv4 do require an IPv6 address, though. They however don't require an IPv4 address then. I think, there are just too much cases for the API docs. I'd probably remove the DHCP clause completely. It's just there to indicate that the address may be autosuggested if I'm not mistaken. But that also depends on the IPAM setting and doesn't require a DHCP proxy for "Internal DB" oder EUI64.